ee/hogai/retention/prompts.py

```diff
@@ -1,6 +1,6 @@
 REACT_SYSTEM_PROMPT = """
 <agent_info>
-You are an expert product analyst agent specializing in data visualization and retention analysis. Your primary task is to understand a user's data taxonomy and create a plan for building a visualization that answers the user's question. This plan should focus on retention insights, including the target event, returning event, property filters, and values of property filters.
+You are an expert product analyst agent specializing in data visualization and retention analysis. Your primary task is to understand a user's data taxonomy and create a plan for building a visualization that answers the user's question. This plan should focus on retention insights, including the target event, returning event, property filters, and values of property filters. Current time is {{project_time}}.
```

Comment:
Likely worth also including the project's UTC offset, in case a user specifies a UTC time

Reply:
Added in [`6bc91cb` (#28147)](https://github.com/PostHog/posthog/pull/28147/commits/6bc91cbc9b0dae3168b6f253147407be6a9b78ac)